### DISCUSSION TITLE
Release candidates

### DISCUSSION THREAD

**Discussion by Pseudomizer** [hassio-addons/addon-unifi#447] [General]
Hello guys,

having upgraded my Dream Machine to 7.5.174 release candidate and trying to install this add-on, it did not work. First it did not my network at all, then I tried to import my site, which is where I found out that this add-on is running 7.4. 

I went into the docker container of this add-on and used the easy upgrade script to upgrade unifi to 7.5, which didn't work. The script went through and installed everything and then got terminated and my bash was killed. 

Upon checking the Add-on, it was still running 7.4. 

Question for the team here: What is the strategy for people who are using release candidate versions as they cannot connect or use this add-on at alll?

Thanks.

**Comment by frenck** [comment]
If you want to run non-stable release, this add-on is not for you.

../Frenck
